src/payment/strategies/klarna/klarna-script-loader.spec.ts

```diff
@@ -11,7 +11,7 @@ describe('KlarnaScriptLoader', () => {
     });
 
     it('loads widget script', () => {
-        klarnaScriptLoader.load();
+        klarnaScriptLoader.loadCredit();
 
         expect(scriptLoader.loadScript).toHaveBeenCalledWith(
             '//credit.klarnacdn.net/lib/v1/api.js'
```

Comment:
You expect this string and in klarna script loader is removed.

Reply:
Actually is not removed I just change the name of the constant